edx_solutions_api_integration/management/commands/add_test_poll_responses.py

```diff
@@ -0,0 +1,185 @@
+"""
+Management command to migrate profile images from s3 bucket to open edx profile images storage
+./manage.py lms migrate_profile_images  --settings=aws --aws-access-key="x" --aws-access-secret="x" --bucket-name="b"
+"""
+
+import logging
+import datetime
+import urllib2 as urllib
+import io
+
+from django.db import connection
+import random
+from itertools import product
+from collections import namedtuple
+
+from optparse import make_option
+from django.core.management.base import BaseCommand, CommandError
+
+from edx_solutions_api_integration.models import APIUser as User
+from course_blocks.api import get_course_blocks
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore import ModuleStoreEnum
+
+from django.contrib.auth.models import User
+from student.models import UserProfile
+from courseware.models import StudentModule
+
+from django.db import transaction
+
+
+# Variables
+
+log = logging.getLogger(__name__)
+
+# Classes
+
+
+class Command(BaseCommand):
+    """
+    Command to generate dummy problem responses xblock poll in a course
+    """
+    help = 'Generates dummy responses for users enrolled in a course'
+    option_list = BaseCommand.option_list + (
+        make_option(
+            "--course-id",
+            dest="course_id",
+            help="Course ID",
+        ),
+        make_option(
+            "--num-responses",
+            dest="num_responses",
+            default=10000,
+            type="int",
+            help="Limit generation of responses to this number. Defaults to "
+                 "generating 10k responses. Setting this to 0 will generate"
+                 "a dummy response for every user in every poll.",
+        ),
+    )
+
+    @staticmethod
+    def get_course_blocks(
+        store,
+        course_key,
+        categories,
+        revision=ModuleStoreEnum.RevisionOption.published_only
+    ):
+        """
+        Retrieve all XBlocks in the course for a particular category.
+
+        By default, only returns only XBlocks that are published.
+        """
+        return [
+            block for block in store.get_items(
+                course_key,
+                qualifiers={"category": categories},
+                revision=revision,
+            )
+        ]
+
+    @staticmethod
+    def get_enrolled_students(course_key):
+        """
+        Given a course_id, returns a QuerySet of all the active students
+        in the course.
+        """
+        return User.objects.filter(
+            courseenrollment__course_id=course_key,
+            courseenrollment__is_active=True,
```

Comment:
Since we are dealing with users created in a test environment, I think it may be OK to generate this for inactive users as well, since test users might be inactive.